desc/grid.py

```diff
@@ -199,6 +209,29 @@ def sym(self):
         """bool: True for stellarator symmetry, False otherwise."""
         return self.__dict__.setdefault("_sym", False)
 
+    @property
+    def is_meshgrid(self):
+        """bool: Whether this grid is a tensor-product grid.
+
+        Let the tuple (r, p, t) ∈ R³ denote a radial, poloidal, and toroidal
+        coordinate value. The is_meshgrid flag denotes whether any coordinate
+        can be iterated over along the relevant axis of the reshaped grid:
+        nodes.reshape(num_radial, num_poloidal, num_toroidal, 3).
+        """
+        return self.__dict__.setdefault("_is_meshgrid", False)
+
+    @property
+    def coordinates(self):
```

Comment:
There are enough coordinates now that it maybe warrants a key in the docstring saying which letter corresponds to what i.e. `a: alpha, the field line label` etc